defaults/main.yml

```diff
@@ -53,6 +53,11 @@ kibana_elasticsearch_pingTimeout:               # Defaults to requestTimeout.
 kibana_elasticsearch_requestTimeout: 30000
 kibana_elasticsearch_shardTimeout: 0
 kibana_elasticsearch_startupTimeout: 5000
+Kibana_elasticsearch_xpack_security: false
+Kibana_elasticsearch_xpack_reporting: true
+Kibana_elasticsearch_xpack_graph: false
+Kibana_elasticsearch_xpack_ml: false
+Kibana_elasticsearch_xpack_monitoring: false
```

Comment:
can these be changed to lower-case 'k' for consistency please

Reply:
This is done now.

Reply:
can they also be changed in templates/kibana-v6.x.yml.j2

Reply:
Updated the kibana-v6.x.yml.j2 as well.